tools/alpine/Makefile

```diff
@@ -2,22 +2,24 @@
 
 ORG?=linuxkit
 IMAGE=alpine
-BASE=alpine:3.6
+
+BASE := $(shell bash -c 'source detect-arch-var && echo $${BASE}')
+DOCKER_CONT := $(shell bash -c 'source detect-arch-var && echo $${DOCKER_CONT}')
 
 default: push
 
 show-tag:
 	@sed -n -e '1s/# \(.*\/.*:[0-9a-f]\{40\}\)/\1/p;q' versions
 
 hash: Dockerfile Makefile packages
-	DOCKER_CONTENT_TRUST=1 docker pull $(BASE)
-	docker build --no-cache -t $(IMAGE):build .
+	DOCKER_CONTENT_TRUST=$(DOCKER_CONT) docker pull $(BASE)
```

Comment:
It would be better to copy the pattern used in `pkg/package.mk` for setting the content trust.